owa/app/js/subscription/subscription.controller.js

```diff
@@ -48,6 +49,22 @@ class SubscriptionController {
         vm.subscription = response.results[0];
       })
     }
+
+    function isVersionAdded(url) {
+      if (url.endsWith("/")) {
+        url = url.substring(0, url.lastIndexOf('/'));
+      }
+      let words = url.split("api.openconceptlab.org")
```

Comment:
Are we intentionally hard coding `api.openconceptlab.org`?

Reply:
So apparently the version section of the URL is not passed as a URL Param and hence extracting it's value it's a little tricky that's why I went the string processing way
However I'm open to suggestions on how to go about this @dkayiwa